### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dominiquejane**
Added some troubleshooting notes to the install section.
Fixed some typos.

### CODE REVIEW

File: golang/dashboard/README.md [hunk 1]
```diff
@@ -38,12 +38,27 @@ $ echo "export PATH=\$PATH:\$GOBIN" >> ~/.bash_profile
...
+```
+
+Don't forget to run:
+```
+source ~/.bash_profile
+```
+
+You may even need to restart your terminal.
+:::
+
```
Comment (ErinHales) [inline golang/dashboard/README.md:56]: Just some formatting suggestions:
```
 > *Troubleshooting tip*
 >
 > If, after following steps 3 and 4 below, you are not able to successfully run `make install` or `make init` then try replacing the above lines with the following:
 >
 >```sh
 > export GOPATH=$HOME/go
 > export PATH=$GOPATH/bin:$PATH
 > ```
 >
 > Don't forget to run:
 > ```sh
 > source ~/.bash_profile
 > ```
 >
 > You may even need to restart your terminal.
```

 - Turns it into a block quote by adding `>` before each line
 - Adds "sh" to code blocks for syntax highlighting
 - Italicizes "Troubleshooting tips" by adding `*`s

Comment (barbaraliau) [reply golang/dashboard/README.md:56]: Does `tip` do the same thing as block quote?

Comment (ErinHales) [reply golang/dashboard/README.md:56]: I'm not sure what `tip` does.  It looks like it wasn't working though 
Screen Shot 2020-04-30 at 3 11 29 PM

Comment (dominiquejane) [reply golang/dashboard/README.md:56]: oh, maybe it's a vuepress thing. I grabbed it off the agoric documentation. hehe...

Comment (ErinHales) [reply golang/dashboard/README.md:56]: Yeah, I just figured out it was Vuepress lol

**Comment by ErinHales** [comment]
Ok, just fix the conflicts, then I think this is good to go :)

**Pull request merged**